### PR TITLE
{bio}[intel/2015b] Sailfish 0.8.0 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/s/Sailfish/Sailfish-0.8.0_skip-included-Jellyfish.patch [hunk 1]
```diff
@@ -0,0 +1,44 @@
+--- sailfish-0.8.0/CMakeLists.txt.orig	2015-11-16 05:16:30.000000000 +0100
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/Sailfish/Sailfish-0.8.0_skip-included-Jellyfish.patch]: Claim ownership and add motivation (although it's quite clear)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/014014b013ca23a510a4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/462553f525279d3e8431 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**